.changeset/smooth-pumpkins-wonder.md

```diff
@@ -0,0 +1,6 @@
+---
+'@twilio-paste/button': patch
+'@twilio-paste/core': minor
```

Comment:
question (non-blocking): why is this a patch for button and a minor for core?

Reply:
DANG - bad copy paste. just pushed a fix.